src/components/experiment-tracking/runs-list-card/runs-list-card.scss

```diff
@@ -17,7 +17,7 @@
 
   &--active {
     background: var(--color-run-list-hover);
-    box-shadow: -1px 0px 0px inset $color-accent-blue;
+    border-right: solid $color-accent-blue;
```

Comment:
The `box-shadow` was used here to prevent a layout shift. If you use a border, you see something like this:

https://user-images.githubusercontent.com/16869061/176423987-463a9d0c-39ad-499e-8dad-d289fc431c3c.mov

You can see the bookmark icon shifting left and right. Try and keep it as a `box-shadow` and see if this still happens when you hover over the card?